assets/sass/components/_frontpage-cards.scss

```diff
@@ -17,6 +19,10 @@
 				font-size: 18px;
 				margin-top: 20px;
 				margin-bottom: 10px;
+
+				a {
+					@include focus-with-background-and-color;
+				}
```

Comment:
And this one I am not a huge fan of, but I guess now that we made only the upper-part of the card clickable, and the footer is not clickable, there is not much choice here:

![Screenshot from 2020-08-17 16-27-22](https://user-images.githubusercontent.com/43036333/90402485-31a13f00-e0a8-11ea-8c00-2583b4abbdec.png)

Reply:
I felt quite the same, but it's important that it outlines the clickable area, so I didn't see much alternative - other ideas are welcome!

Reply:
I think it works well, when there are clickable buttons in the bottom, which will be focused before moving to the next card.